app/models/podcast_import.rb

```diff
@@ -114,10 +114,6 @@ def import_episodes!
     # Request the RSS feed again
     get_feed
 
-    # Update podcast attributes
-    create_or_update_podcast!
```

Comment:
this method needed to be removed here since it defeats the purpose of the above-mentioned boolean for `import_podcast!`; it's already called in that method.

Reply:
Huh, weird.  It does seem a bit redundant/out-of-place.  Good call!